model/qti/ImportService.php

```diff
@@ -495,8 +486,7 @@ public function importQtiItem(
                             \common_Logger::i(
                                 'Resource "' . $resourceIdentifier . '" is already stored in the database and will not be imported.'
                             );
-                            $this->checkImportLockTime($startImportTime, $qtiItemResource->getIdentifier());
-                            $lock->release();
```

Comment:
Why have you deleted all releasing methods? We do need to release item on import finished, other processes that wait for this item will be blocked